modules/Bio/EnsEMBL/Analysis/Hive/Config/Projection_subpipeline_conf.pm

```diff
@@ -0,0 +1,518 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Analysis::Hive::Config::Projection_subpipeline_conf;
+
+use strict;
+use warnings;
+use File::Spec::Functions;
+
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+use Bio::EnsEMBL::Analysis::Tools::Utilities qw(get_analysis_settings);
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use base ('Bio::EnsEMBL::Analysis::Hive::Config::HiveBaseConfig_conf');
+
+sub default_options {
+  my ($self) = @_;
+  return {
+    # inherit other stuff from the base class
+    %{ $self->SUPER::default_options() },
+
+######################################################
+#
+# Variable settings- You change these!!!
+#
+######################################################
+########################
+# Misc setup info
+########################
+    'dbowner'                   => '' || $ENV{EHIVE_USER} || $ENV{USER},
+    'pipeline_name'             => '' || $self->o('production_name').'_'.$self->o('ensembl_release'),
+    'production_name'           => '', # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'release_number'            => '' || $self->o('ensembl_release'),
+    'user_r'                    => '', # read only db user
+    'user'                      => '', # write db user
+    'password'                  => '', # password for write db user
+    'pipe_db_server'            => '', # host for pipe db
+    'dna_db_server'             => '', # host for dna db
+    'databases_server'          => '', # host for general output dbs
+    'pipe_db_port'              => '', # port for pipeline host
+    'dna_db_port'               => '', # port for dna db host
+    'databases_port'            => '', # port for general output db host
+    'uniprot_set'               => '', # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'               => '', # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'use_genome_flatfile'       => '1',# This will read sequence where possible from a dumped flatfile instead of the core db
```

Comment:
Strangely enough, it doesn't seem to be use, so it can be removed

Reply:
Removed